src/index.tsx

```diff
@@ -83,8 +83,8 @@ export class Piece extends React.Component<Props> {
   }
 
   render () {
-    const { avatarStyle, style, pieceType, pieceSize } = this.props
-    return <PieceComponent avatarStyle={avatarStyle as AvatarStyle} style={style} pieceType={pieceType} pieceSize={pieceSize}/>
+    const { avatarStyle, style, pieceType, pieceSize, viewBox } = this.props
```

Comment:
I guess you may also want to add viewBox to the property type?

Reply:
Yeah I forgot! Just add viewBox to property type. Thanks!